examples/getopt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-getopt)
+
+hunter_add_package(getopt)
+find_package(getopt CONFIG REQUIRED)
+
+add_executable(getopt_example main.cpp)
```

Comment:
Had planned to just remove all the others from the test, but handling it this way should verify that getopt exist elsewhere.